### PR TITLE
Mathfont package option

### LINKED ISSUE #11: Default math font aestethics
**Personal opinion**: The default font looks terrible in math mode, and anyone who uses this slide template in a presentation should be aware of this. 

Reference pictures with different math fonts. Default (top), mathserif (bottom)
before
after

### PR DESCRIPTION
**Pull request by kklocker**
Fixes #11 . 

This seem to work just fine, although it would be "prettier" to not have to pass mathfont=serif as default. I have not managed to do that, but suggestions for improvements are welcome -- I have never done this before :))

### CODE REVIEW

File: beamerthemeNTNU.sty [hunk 1]
```diff
@@ -85,15 +85,28 @@
 \DeclareOptionBeamer{hideframetotal}{%
 	\ntnu@displayTotalFrameNumberfalse%
 }
+
+\DeclareOptionBeamer{mathfont}[sans]{
+	
+	\ifnum\pdf@strcmp{#1}{sans}=\z@%
+		\usefonttheme{structurebold}%
```
Comment (kellertuer) [inline beamerthemeNTNU.sty]: Why did you move the structure bold herein?

**Review by kellertuer — COMMENTED** [review]
Thanks for the PR!

For the default I would like to stay non-breaking, i.e. that sans is the default.

For the theme examples you do not have to add that option, but for the nor/eng examples that was exactly what I had in mind.

It would be great if you could add this option to the documentation as well.

File: beamerthemeNTNU.sty (hunk 1 above)
Comment (kklocker) [reply beamerthemeNTNU.sty]: > Why did you move the structure bold herein?

Earlier attempts of  solutions that didn't work... I will revert this when i fix the rest :)

**Comment by kklocker** [comment]
I think at least one of the examples included in the documentation should display the effect of setting `mathfont=serif`, by adding a sentence on the slides ~"Math mode looks like this: $E = mc^2$"

**Comment by kellertuer** [comment]
The update looks already quite good, I will check the next days that it works as I hope it does (from reading the code). Thanks for also providing an update to the docs; my only meta-critic would be that the math looks a little boring and something with an integral would be sweet, too. But that is of course just a personal opinion; technically this already looks very good.

Thanks for your work.

**Comment by kklocker** [comment]
Done :)

**Comment by kellertuer** [comment]
I think this is great and can be merged. Could you just replace
`2021/05/26 v1.0` with `2021/06/28 v1.1` then (just a global search/replace)?
This will make it easier when requests/bugs are found or need to be investigated.

**Comment by kklocker** [comment]
I pushed a tag v1.1 to my local repo, i don't know if this will appear in this repo as well?

**Comment by kellertuer** [comment]
Yes, that is synced – your local branch (oh you even worked on your master) and this one here. Perfect. Then we can merge.
